lib/task.js

```diff
@@ -221,11 +223,12 @@ export class Reclaimer {
     this.log('reclaiming task');
 
     try {
-      this.claim = await this.runtime.queue.reclaimTask(
+      this.claim = await this.task.queue.reclaimTask(
```

Comment:
We should have a queue client per Reclaimer because the creds will be different per task we're claiming.  This happens when there are coalesced tasks where there is the primary task we're executing, but we need to reclaim and resolve each individual task that was coalesced.